Movapp Widget/Movapp_Widget.swift

```diff
@@ -0,0 +1,123 @@
+import WidgetKit
+import SwiftUI
+import Intents
+
+struct Provider: IntentTimelineProvider {
+    
+    let store = DictionaryDataStore()
+    
+    init() {
+        store.load(language: .csUk)
+    }
+    
+    func placeholder(in context: Context) -> FavoritePhraseEntry {
+        FavoritePhraseEntry.example()
+    }
+
+    func getSnapshot(for configuration: ConfigurationIntent, in context: Context, completion: @escaping (FavoritePhraseEntry) -> ()) {
+        completion(FavoritePhraseEntry.example())
+    }
+
+    func getTimeline(for configuration: ConfigurationIntent, in context: Context, completion: @escaping (Timeline<Entry>) -> ()) {
+        
+        guard let phrases = store.dictionary?.translations.values else {
+            let timeline = Timeline(entries: [FavoritePhraseEntry.example(intent: configuration)], policy: .atEnd)
+            return completion(timeline)
+        }
+        
+        var entries: [FavoritePhraseEntry] = []
```

Comment:
Nejde FavoritePhraseEntry zjednodušit?

struct FavoritePhraseEntry {

translation: Translation
date,
configuration

?